src/cryptonote_basic/cryptonote_format_utils.h

```diff
@@ -300,14 +300,14 @@ constexpr std::string_view get_unit() {
 // Returns a monetary value with a decimal point; optionally strips insignificant trailing 0s.
 std::string print_money(
         uint64_t amount,
-        size_t decimal_places = oxen::DISPLAY_DECIMAL_POINT,
-        bool strip_zeros = false);
+        bool strip_zeros = false,
+        size_t decimal_point = oxen::DISPLAY_DECIMAL_POINT);
```

Comment:
There are some uses of this in the codebase that are now passing reverse arguments here (I think they may be recent dev additions).  Adding these here would make those errors:
```C++
// Guard against passing arguments in wrong order:
std::string print_money(uint64_t, size_t, bool) = delete;
std::string print_money(uint64_t, size_t) = delete;
```